### PR TITLE
fix: when a query returns no results, some buttons needs to be greyed out

### PR DESCRIPTION
**Pull request by muhammadjufry**
Closes: #4959
### Description:
What i have fixed:
- [X] The Bar chart icon needs to be greyed out
- [X] The export Csv button needs to be greyed out
- [X] There is no need to show the tiny No results label below the table as it is already implied by Query returns no results

Here's the video demo: https://www.awesomescreenshot.com/video/16292404?key=80d64dc4e50160047560de8a64d83dcd
Many thanks

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -35,4 +35,4 @@ ALLOW_MULTIPLE_ORGS=false
 
 DBT_PROJECT_DIR=.
 
-SCHEDULER_ENABLED=true
+SCHEDULER_ENABLED=true
```
Comment (IrakliJani) [inline .env:38]: this is an unnecessary change. can your revert?

File: packages/frontend/src/components/Explorer/VisualizationCardOptions/index.tsx [hunk 2]
```diff
@@ -46,7 +46,14 @@ const VisualizationCardOptions: FC = memo(() => {
                     return {
                         text: 'Mixed',
                         icon: (
-                            <IconChartAreaLine size={20} color={Colors.GRAY1} />
+                            <IconChartAreaLine
+                                size={20}
+                                color={
+                                    disabled === true
```
Comment (IrakliJani) [inline packages/frontend/src/components/Explorer/VisualizationCardOptions/index.tsx:52]: you can just have `disabled ? Colors.LIGHT_GRAY1 : Colors.GRAY1`

File: packages/frontend/src/components/Explorer/VisualizationCardOptions/index.tsx [hunk 3]
```diff
@@ -57,15 +64,29 @@ const VisualizationCardOptions: FC = memo(() => {
                         return {
                             text: 'Area chart',
                             icon: (
-                                <IconChartArea size={20} color={Colors.GRAY1} />
+                                <IconChartArea
+                                    size={20}
+                                    color={
+                                        disabled === true
```
Comment (IrakliJani) [inline packages/frontend/src/components/Explorer/VisualizationCardOptions/index.tsx:70]: same here

Comment (IrakliJani) [reply packages/frontend/src/components/Explorer/VisualizationCardOptions/index.tsx:70]: and basically everywhere you have `disabled === true`

**Review by IrakliJani — COMMENTED** [review]
looks great @muhammadjufry added minor comments. lmk when it's ready and I'll run tests → approve → merge 🙌

File: packages/frontend/src/components/Explorer/VisualizationCardOptions/index.tsx (hunk 2 above)
Comment (muhammadjufry) [reply packages/frontend/src/components/Explorer/VisualizationCardOptions/index.tsx:52]: ok sure

File: .env (hunk 1 above)
Comment (muhammadjufry) [reply .env:38]: sure

File: packages/frontend/src/components/Explorer/VisualizationCardOptions/index.tsx (hunk 3 above)
Comment (muhammadjufry) [reply packages/frontend/src/components/Explorer/VisualizationCardOptions/index.tsx:70]: ok sure

File: .env (hunk 1 above)
Comment (muhammadjufry) [reply .env]: sure

**Comment by IrakliJani** [comment]
hi @muhammadjufry everything looks great, tested and everything. but we have a failing test

File is `downloadCsv.cy.ts:102:11`
here is a copy of it:

```
  Download CSV on Explore
    (Attempt 1 of 2) Should download CSV from results on Explore
    1) Should download CSV from results on Explore
    ✓ Should download CSV from table chart on Explore (5268ms)


  3 passing (37s)
  1 failing

  1) Download CSV on Explore
       Should download CSV from results on Explore:
     AssertionError: Timed out retrying after 5000ms: Expected to find element: `[data-icon="export"]`, but never found it.
      at Context.eval (webpack:///./cypress/e2e/downloadCsv.cy.ts:102:11)
```

I can push a fix for you or lmk if you need any help.

you can click here to see the test output and video attached:

Screenshot 2023-04-12 at 17 22 14

**Comment by muhammadjufry** [comment]
Thank you

**Comment by muhammadjufry** [comment]
Hi @IrakliJani How is it going?

**Comment by muhammadjufry** [comment]
ok no problem, thank you

**Review by IrakliJani — APPROVED** [review]

**Comment by IrakliJani** [comment]
@all-contributors add @muhammadjufry for bug

**Comment by IrakliJani** [comment]
@muhammadjufry sorry for the late approval, we've been having some issues regarding CSV testing

**Comment by muhammadjufry** [comment]
> @muhammadjufry sorry for the late approval, we've been having some issues regarding CSV testing
No problem, thank you
